index.js

```diff
@@ -14,17 +40,15 @@ module.exports = {
 
     this.theme = theme;
 
-    if (!process.env.EMBER_CLI_FASTBOOT) {
-      this.app.import('vendor/jquery-disablescroll/jquery.disablescroll.js');
-    }
+    this.app.import('vendor/jquery.disablescroll.js');
 
     this._super.included.apply(this, arguments);
   },
+
   options: {
     nodeAssets: {
       'tether-shepherd': function() {
         return {
-          enabled: !process.env.EMBER_CLI_FASTBOOT,
```

Comment:
You have to also use the fastboot transform with processTree in node assets. Something like this https://github.com/shipshapecode/ember-flatpickr/blob/master/index.js#L20